codecov.yml

```
@@ -0,0 +1,11 @@
+coverage:
+  # Tell codecov our default branch.
+  branch: main
+  # Don't trigger CI failure on coverage reduction.
+  status:
+    project:
+      default:
+        informational: true
+    patch:
+      default:
+        informational: true
```

nit: missing newline
```suggestion
        informational: true
        
```